@here/harp-mapview/lib/Tile.ts

```diff
@@ -1332,7 +1362,9 @@ export class Tile implements CachedResource {
 
                 if (
                     !bufferGeometry.getAttribute("normal") &&
-                    (isStandardTechnique(technique) || isStandardTexturedTechnique(technique))
+                    (isStandardTechnique(technique) ||
+                        isStandardTexturedTechnique(technique) ||
+                        isStandardTexturedTerrainTechnique(technique))
```

Comment:
For terrain we actually don't need per vertex normals.

Reply:
True, will remove.

Reply:
Actually, we do need to compute the normals since they're used in the vertex-shader displacement. Readded it.

Reply:
hm, I see. Another reason to have custom shader code for the displacement, b/c the normals will all be (0,0,1).
But maybe having per vertex normals is more future proof when thinking about the globe.